msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.Common.targets

```diff
@@ -536,6 +536,9 @@ Copyright (C) 2014 Xamarin. All rights reserved.
 	<Target Name="_CompileToNative" DependsOnTargets="_DetectAppManifest;_DetectSdkLocations;_GenerateBundleName;ResolveReferences;_CompileEntitlements;_CompileAppManifest"
 		Inputs="$(TargetDir)$(TargetFileName)"
 		Outputs="$(_AppBundlePath)Contents\MacOS\$(_AppBundleName)">
+		<ItemGroup>
+			<ReferencesOnly Include="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.Extension)' == '.dll'" />
```

Comment:
Maybe name like `ReferenceCopyLocalAssemblyPaths`?

Reply:
Wouldn't that just append to the existing item group of that name?

Reply:
oh, woops, the existing name doesn't have the word Assembly in it. n/m :)

Reply:
yeah, I just wanted to make it easy to see where that extracted item was sourced from!